Create a language selection demo

jerrylususu opened the pull request:
I am working on a project that will be used by users speaking different languages. PySimpleGUI is a great tool, but it seems that there isn't much resource about internationalization (i18n) for PySimpleGUI. Therefore I created a simple example, in which the program starts with a language selection window, and then load the main window with all the strings in the selected language. It works pretty well for my use case. Still, I am fully aware that this technique won't allow changing language on the fly, and I am still trying to find how to make that work.

Here is a screenshot.
![image](https://user-images.githubusercontent.com/17522475/174823245-61c46e1f-0b6c-4a6b-ac75-0050a07434a4.png)

PySimpleGUI commented:
Hi @jerrylususu 
Thank you for taking the time to put together a demo.
The PySimpleGUI project doesn't accept pull requests as explained in the contributing.md and documentation.
However, I would like for you to do a couple of things with your demo:
1. Make your code available in a repo in your account
2.  Post your screenshots in Issue #10 along with a link to your repo so that others can see what you've made as well as where to get the code.
3. Optionally, also make an entry in the PySimpleGUI Wiki located in this GitHub with a link to your repo and the screenshot.  It's a bit more involved than posting in Issue 10, but it's more permanent in some ways.

I like what you've done and don't want to see your effort not used.  
The PySimpleGUI is a little unusual with this policy and it's most certainly nothing specific to your submission.... I would like you to make it available.  Please let me know when you make it available and I'll definitely visit your repo and give it a * so others will be informed.

Thank you for your effort!  ![image](https://user-images.githubusercontent.com/46163555/174883881-41292c6a-2297-428d-8063-a69bd90851ee.png)

PySimpleGUI commented:
Your code did give me an idea of creating a section in the eCookbook for user submissions.

So I created one with your code.

Very strangely, it is not showing the strings that are not in English.

Can you take a look?  Do you have any idea why this may be the case?
https://trinket.io/pygame/230a32be16

I'll contact support at Trinket and ask them as well.  

I think a User Submissions section of the eCookbook would be a nice addition and would like to add your program as the first entry.

jerrylususu commented:
> Very strangely, it is not showing the strings that are not in English.

One possible reason might be that the virtual environment used by trinket doesn't have Chinese fonts installed. Unlike English and most languages, Chinese has a very large character set, resulting in large font files. Therefore, it is a common practice for English-oriented Linux installs to not include Chinese font files, so nothing will appear. In this specific case, replacing Chinese with more widely supported languages, such as Spanish or French might help.

jerrylususu commented:
Hi! Sorry I am busy with other things recently. I have 
- published my code in a public repo: https://github.com/jerrylususu/PySimpleGUI-Demo-LangSelect
- added a reply to the screenshot issue: https://github.com/PySimpleGUI/PySimpleGUI/issues/10#issuecomment-1164384709
- created a section of my demo in wiki: https://github.com/PySimpleGUI/PySimpleGUI/wiki#language-selection-for-simple-i18n

By the way, I added a French translation, which seems to work in trinket.io. However as I am in a free plan it seems that I can not share a link with you.

![image](https://user-images.githubusercontent.com/17522475/175307769-9f652b03-c077-420d-bd5c-e6cfb4e41746.png)
